src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2023,11 +2017,11 @@ From there, simply click on one of the supported files in the File Explorer and
     <value>Customize the shortcut to pin or unpin an app window</value>
     <comment>"Always On Top" is the name of the utility</comment>
   </data>
-  <data name="Oobe_AlwaysOnTop" xml:space="preserve">
+  <data name="Oobe_AlwaysOnTop.Title" xml:space="preserve">
     <value>Always On Top</value>
     <comment>"Always On Top" is the name of the utility</comment>
   </data>
-  <data name="Oobe_AlwaysOnTop_Description" xml:space="preserve">
+  <data name="Oobe_AlwaysOnTop.Description" xml:space="preserve">
```

Comment:
Don't these changes in Resources.resw mean we'll lose localization for this release until loc team has the chance to catch up?